etc/profile-a-l/com.github.tchx84.Flatseal.profile

```diff
@@ -0,0 +1,62 @@
+# Firejail profile for flatseal
+# This file is overwritten after every install/update
+# Persistent local customizations
+include com.github.tchx84.Flatseal.local
+# Persistent global definitions
+include globals.local
+
+# Reading dconf is required for GTK theme to work.
+noblacklist ${HOME}/.config/dconf/
+noblacklist ${HOME}/.local/share/flatpak/overrides
+noblacklist /var/lib/flatpak/app
+
+include allow-gjs.inc
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-programs.inc
+include disable-shell.inc
+include disable-proc.inc
+include disable-xdg.inc
+
+mkdir ${HOME}/.local/share/flatpak/overrides
+whitelist ${HOME}/.config/dconf/
+whitelist ${HOME}/.local/share/flatpak/overrides
+
+apparmor
+caps.drop all
+net none
+no3d
+nodvd
+nogroups
+noinput
+nonewprivs
+noroot
+nosound
+notv
+nou2f
+novideo
+protocol unix
+seccomp
+seccomp.block-secondary
+shell none
+tracelog
+
+disable-mnt
+private-bin com.github.tchx84.Flatseal,gjs
+private-cache
+private-dev
+private-etc alternatives,dconf,fonts,gtk-3.0,ld.so.cache,ld.so.preload
+private-tmp
+
+dbus-user filter
+dbus-user.own com.github.tchx84.Flatseal
+dbus-user.talk ca.desrt.dconf
+dbus-user.talk org.freedesktop.impl.portal.PermissionStore
+dbus-user.talk org.gnome.Software
+dbus-system none
+
+read-only ${HOME}/.config/dconf/
+read-write ${HOME}/.local/share/flatpak/overrides
```

Comment:
Never `read-only`ed

Reply:
Removing this line, I get an error:

```
(com.github.tchx84.Flatseal:12): Gjs-CRITICAL **: 16:20:46.526: JS ERROR: GLib.FileError: Failed to create file “/home/hugo/.local/share/flatpak/overrides/com.github.Eloston.UngoogledChromium.XYTID1”: Read-only file system
_saveOverrides@resource:///com/github/tchx84/Flatseal/js/models/permissions.js:182:21
_updateModels@resource:///com/github/tchx84/Flatseal/js/models/permissions.js:222:14
main@resource:///com/github/tchx84/Flatseal/js/main.js:34:24
run@resource:///org/gnome/gjs/modules/script/package.js:206:19
@/usr/local/bin/com.github.tchx84.Flatseal:9:17
```

Reply:
Interesting, `${HOME}/.local/share/flatpak/exports` is the only `read-only`ed dir in `~/.local/share/flatpak`. What could cause this?

Reply:
It's set read-only in `disable-common.inc`.

Reply:
`flatpak/exports` yes but `flatpak/overrides`?

```
$ grep "read-only .*flatpak" /etc/firejail/*.inc
/etc/firejail/disable-common.inc:read-only ${HOME}/.local/share/flatpak/exports
```

Reply:
Somewhat out-of-scope: what do you think about whitelisted locations being read-only by default, and requiring that read-write be set explicitly?

Currently, a lot of locations are whitelisted, and all those are made read-write implicitly, which really increases the scope of how much access each sandbox has (and how much impact they can have messing around).

Reply:
> Somewhat out-of-scope: what do you think about whitelisted locations being read-only by default, and requiring that read-write be set explicitly?

Generally I thinks it's a bad idea.
1. It's a not small breaking change used in hundreds of profiles and locals.
2. It's not what users expect if they `whitelist ${HOME}/my-documents` in a local.

In specific of wc, we could add `read-only` all `whitelist`ed paths (which don't need to be read-write).

Reply:
True, it'd be a mess in terms of compat and breaking people's overrides too.

I think `whitelist-ro` makes sense too. My train of though is mostly that, if I want to grant just read-only access to a directory right now, I need to `whitelist`, which grants write access _and then_ remove the write access, which just sounds... not right. I'll experiment a bit with that and see if I can produce a viable PR.

Reply:
Maybe it's just a `profile_add("whitelist {path}"); profile_add("read-only {path}");`, IDK.